source/documentation/common/install/proc-Deploying_the_Self-Hosted_Engine_Using_the_CLI.adoc

```diff
@@ -189,7 +189,7 @@ SSH public key []:
 Do you want to enable ssh access for the root user (yes, no, without-password) [yes]:
 ----
 
-. Optional: You can apply the DISA STIG security profile on the {engine-name} virtual machine. The DISA STIG profile is the default OpenSCAP profile. For more information, see xref:disa-stig-for-red-hat-linux_assembly-securing-rhv[]
+. Optional: You can apply the DISA STIG security profile on the {engine-name} virtual machine. The DISA STIG profile is the default OpenSCAP profile.
```

Comment:
Was removing the xref here intentional?

Reply:
Yes. It only applies to downstream. It has to be conditionalized because the link does not work in upstream version because the assembly is not included in ovirt. I guess no one noticed.

Reply:
This is a good argument for the practice of putting xrefs only in assemblies. It is a lot easier to conditionalize sections of assemblies than in modules.